feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/vif.c

```diff
@@ -1317,11 +1335,11 @@ bool target_vif_config_set2(const struct schema_Wifi_VIF_Config *vconf,
 			const struct schema_Wifi_VIF_Config_flags *changed,
 			int num_cconfs)
 {
-
+	int rc;
 	if (!strcmp(vconf->mode, "mesh")) {
-		mesh_vif_config_set(rconf, vconf, changed);
+		rc = mesh_vif_config_set(rconf, vconf, changed);
 	} else {
-		ap_vif_config_set(rconf, vconf, changed);
+		rc = ap_vif_config_set(rconf, vconf, changed);
```

Comment:
Return is not checked. Should we check and log failure case?

Reply:
I generated a log at the point of detection.  Passed the error up the chain, but there is nothing useful to be done beyond this function.